ansible/roles/isic/vars/main.yml

```diff
@@ -1,8 +1,7 @@
 ---
 
-girder_version: "933096b33334bb9c30c3c7a659dbab32691cba3d"
-worker_version: "042a69d6ae7d472e32242d98f8b7acb3148980f9"
-large_image_version: "4dc82976d58b2a6e6542863e2e410628649a040c"
+girder_version: "master"
```

Comment:
Do either of these two variables get used anymore? Both should be indirectly installed by `pip install isic-archive`.